src/util/util.go

```diff
@@ -67,11 +67,10 @@ func RunCommandAndLog(cmd *exec.Cmd) string {
 	cmd.Stdout = &out
 	cmd.Stderr = &stderr
 	err := cmd.Run()
+	log.Infof("Command result: %s", out.String())
 	if err != nil {
-		log.Infof("Command result: %s", out.String())
 		log.Infof("%s: %s", fmt.Sprint(err.Error()), stderr.String())
 	}
```

Comment:
Seems like the e2e test passed despite failing because of this part. `log.Fatalf` was changed to `log.Infof` during the sls deploy debugging and it got merged: https://github.com/vhive-serverless/STeLLAR/commit/77b940cddb82555f62531ece434535e8bfc6f5e1

Reverting back to `log.Fatalf("%s: %s", fmt.Sprint(err.Error()), stderr.String())` should give us the expected failure if the deployment service fails after this.

Reply:
Thanks Min, I've made the change back to `log.Fatalf`